src/main/scala/mesosphere/marathon/api/serialization/ContainerSerializer.scala

```diff
@@ -419,6 +435,50 @@ object MesosDockerSerializer {
   }
 }
 
+object LinuxInfoSerializer {
+  def fromProto(proto: Protos.ExtendedContainerInfo.LinuxInfo): Option[LinuxInfo] = {
+    if (!proto.hasSeccomp) return None
```

Comment:
Can we avoid `return None` here? See https://tpolecat.github.io/2014/05/09/return.html for more info.